src/LMSModule.php

```diff
@@ -5,7 +5,8 @@
 class LMSModule 
 {
     protected $protected_tables = [
-        'lms_users',
+        'course_user',
+        'courses',
```

Comment:
just to make sure, is it just the table name without the badaso prefix?

Reply:
i'll double check

Reply:
yep double checked, its true without the prefix